gs.js

```diff
@@ -86,44 +101,43 @@ class GlobalStorage extends lib.StorageProvider {
   }
 
   // Create new object
+  //   category - <string>, category to store the object in
   //   obj - <Object>
   //   callback - <Function>
   //     err - <Error> | <null>
   //     id - <string>
-  create(obj, callback) {
-    this.local.create(obj, callback);
+  create(category, obj, callback) {
+    this.local.create(category, obj, callback);
   }
 
   // Update Object
-  //   obj - <Object>
+  //   category - <string>, category to update the records in
+  //   query - <Object>, example: { Id }
+  //   patch - <Object>, fields to update
   //   callback - <Function>
   //     err - <Error> | <null>
-  update(obj, callback) {
-    this.local.update(obj, callback);
+  //     count - <number>
+  update(category, query, patch, callback) {
+    this.local.update(category, query, patch, callback);
   }
 
   // Delete object
-  //   id - <string>, object id
+  //   category - <string>, category to delete the records from
+  //   query - <Object>, example: { Id }
   //   callback - <Function>
   //     err - <Error> | <null>
-  delete(id, callback) {
-    this.local.delete(id, callback);
+  //     count - <number>
+  delete(category, query, callback) {
+    this.local.delete(category, query, callback);
   }
 
   // Select dataset
-  //   query - <Object>
-  //   options - <Object>
-  //   callback - <Function>
-  //     err - <Error> | <null>
-  //     cursor - <MemoryCursor>
-  select(query, options, callback) {
-    return this.local.select(query, options, callback);
-  }
-
-  // Get server for id
-  //   id - <string>, object id
-  findServer(id) {
-    return id;
+  //   category - <string>, category to select the records from
+  //   query - <Object>, fields conditions
+  //
+  // Returns: <Cursor>
+  select(category, query) {
+    return this.local.select(category, query);
   }
 
   // Get server for id
```

Comment:
I think this comment is incorrect.

Reply:
Should we maybe remove this method completely?